examples/react/getting-started/src/App.tsx

```diff
@@ -21,7 +22,7 @@ import './App.css';
 
 const searchClient = algoliasearch(
   'latency',
-  '6be0576ff61c053d5f9a3225e2a90f76'
+  '312a11382a63bdfc06e3183492fc469f'
```

Comment:
why is the api key changed? does the original one not get access to agents / this agent?

Reply:
I was experimenting and forgot to change it back